cli.js

```diff
@@ -947,7 +947,7 @@ program
     .description('Run an instance export')
     .action(function(options) {
```

Comment:
Please adjust the description of option `-d,--data <data>`

Reply:
Hello @tobiaslohr , I updated it. Let me know if this works or if you want me to modify anything else or if you had something else in mind.

Reply:
hmm. I grabbed the latest changes that were merged into master into my branch and the circle ci build is failing but it won't let me rerun it. I think that only had changes to the README.md so it shouldn't have really affected anything. Is there any way it can be rerun?